src/core/custom_tex_cache.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <unordered_map>
+#include <vector>
+#include "common/common_types.h"
+
+namespace Core {
+struct CustomTexInfo {
+    u32 width;
+    u32 height;
+    std::vector<u8> tex;
+};
+
+// TODO: think of a better name for this class...
+class CustomTexCache {
+public:
+    CustomTexCache();
+    ~CustomTexCache();
+
+    bool IsTextureDumped(u64 hash) const;
+    void SetTextureDumped(u64 hash);
+
+    bool IsTextureCached(u64 hash) const;
+    const CustomTexInfo& LookupTexture(const u64 hash);
```

Comment:
```suggestion
    const CustomTexInfo& LookupTexture(u64 hash) const;
```
1. `const` qualified parameter in declaration is useless (plus it is quite meaningless even if in definition)
1. this can be a const member function